test/envoye2e/driver/envoy.go

```diff
@@ -26,6 +26,10 @@ import (
 	core "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
 	v2 "github.com/envoyproxy/go-control-plane/envoy/config/bootstrap/v2"
 
+	// Preload proto definitions
+	_ "github.com/cncf/udpa/go/udpa/type/v1"
+	_ "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/http_connection_manager/v3"
```

Comment:
We should not need this if we use udpa struct right?

Reply:
Yes, but we should only use UDPA struct for open-ended config, like Wasm since it keeps changing. It's ok to use Any for stable configs.